src/composables/balances.ts

```diff
@@ -45,6 +47,21 @@ export function useBalances({ store }: IDefaultComposableOptions) {
       .toFixed(),
   );
 
+  const balancesCurrenciesTotal = computed(
+    () => {
+      const result = Object.entries(balances.value)
```

Comment:
I feel that we are doing the reversed logic :)
So what actually we need to achieve is to sum ap all the **accounts** balances. So I suggest to loop over the accounts and try to collect the balances for them. This way you won't need to search account in every iteration.